build.cake

```diff
@@ -180,7 +172,10 @@ Task("Test")
 	.IsDependentOn("Build")
 	.Does(() =>
 	{
-		NUnit3(BIN_DIR + UNIT_TEST_ASSEMBLY);
+		foreach(var framework in TARGET_FRAMEWORKS)
+		{
+			NUnit3(System.IO.Path.Combine(BIN_DIR, framework, UNIT_TEST_ASSEMBLY));
+		}
```

Comment:
I wanted to do this by passing the solution file, but nunit/vs-project-loader#26 😞